### DISCUSSION TITLE
How do I set the surface material for the style of an object in the geometrylayer

### DISCUSSION THREAD

**Discussion by yangcumtb** [iTowns/itowns#2380] [Ideas]
How do I load a local json file in my application in the format of geojson features？
This is an example of GeoJSON to 3D objects in the sample：[](https://www.itowns-project.org/itowns/examples/#source_file_geojson_3d)

I noticed that he only set the fill color of the style, but did not set the surface material of the object. How can I achieve this effect?

If I don't assign a value to fill.color in style, it defaults to this
<img width="593" alt="image" src="https://github.com/user-attachments/assets/0b404aa3-ddd8-48c1-8969-da313d86aa79">

Should this surface effect be the default material effect for three? If so, is there a way to use ThreeJS to update surface material for better visualization?

**Comment by jailln** [comment]
Hi, can you provide a jsfiddle with your code please ?
